scriptmodules/ports/devilutionx.sh

```diff
@@ -12,7 +12,7 @@
 
 rp_module_id="devilutionx"
 rp_module_desc="devilutionx - Diablo Engine"
-rp_module_licence="https://raw.githubusercontent.com/diasurgical/devilutionX/master/LICENSE"
+rp_module_licence="SUL1 https://github.com/diasurgical/devilutionX/blob/master/LICENSE.md"
```

Comment:
Most times they use the raw url: https://raw.githubusercontent.com/diasurgical/devilutionX/master/LICENSE.md